compiler/plugins/target/ROCM/Dialect/ROCM/Transforms/test/apply_builtin_ukernel_pdl_patterns.mlir

```diff
@@ -18,19 +22,19 @@ func.func @matmul_f8(%arg0: tensor<1x128x4096xf8E4M3FNUZ>, %arg1: tensor<1024x40
     } -> tensor<1x128x1024xf32>
   return %2 : tensor<1x128x1024xf32>
 }
-// CHECK-LABEL: @matmul_f8
-// CHECK:         linalg.generic
-// CHECK-SAME:      compilation_info = #iree_codegen.compilation_info
-// CHECK-SAME:      lowering_config =
-// CHECK-SAME:      translation_info =
-// CHECK-SAME:      iree_codegen.ukernel = #iree_codegen.ukernel_descriptor<"pingpong_medium_f8_expanded", tensor>
+// GFX942-LABEL: @matmul_f8_gfx942
+// GFX942:         linalg.generic
+// GFX942-SAME:      compilation_info = #iree_codegen.compilation_info
+// GFX942-SAME:      lowering_config =
+// GFX942-SAME:      translation_info =
+// GFX942-SAME:      iree_codegen.ukernel = #iree_codegen.ukernel_descriptor<"pingpong_medium_f8E4M3FNUZ_expanded", tensor>
```

Comment:
I think it would be useful to add the negative version for gfx950 after this to ensure that one doesn't match: 
```
GFX950-NOT: iree_codegen.ukernel = #iree_codegen.ukernel_descriptor
```
Same for the other ones below.